services/optimiser/input_processing.py

```diff
@@ -349,7 +349,7 @@ def get_position_qualification(session, position_id):
     return qualification_id
 
 
-# This can be called by api from postman, to test easily
+# This can be called by api from postman, to tests easily
```

Comment:
Please move this into the test directory. I know you didn't write it, but I would appreciate it.